ocs_ci/ocs/ui/base_ui.py

```diff
@@ -856,7 +858,7 @@ def login_ui(console_url=None):
 
         # headless browsers are web browsers without a GUI
         headless = ocsci_config.UI_SELENIUM.get("headless")
-        if headless:
+        if not headless:
```

Comment:
Please don't add 'not headless' to pr because we run the test on jslave... We use it only for debugging.

Reply:
done